ocs_ci/ocs/must_gather/must_gather.py

```
@@ -142,20 +142,17 @@ def search_file_path(self):
-                file_path = os.path.join(path, file)
-                if (
-                    Path(file_path).stat().st_size == 0
-                    and "noobaa-db-pg-0-init.log" not in file_path
-                ):
-                    logger.error(f"log file {file} empty!")
-                    self.empty_files.append(file)
+        for file, file_path in self.files_path.items():
+            if not os.path.isabs(file_path):
+                continue
+            if Path(file_path).stat().st_size == 0:
+                logger.error(f"log file {file} empty!")
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)